### DISCUSSION TITLE
Bugs in preview

### DISCUSSION THREAD

**Discussion by IeuanWalker** [dotnet/maui#3704] [General] [answered]
Hi, just wondering if we should be raising bugs found in the MAUI preview, or if because it is still under active development should wait for RC version?

**Comment by nogginbox** [comment, marked as answer]
You  can and should raise bugs here in Github. The main reason to do development in the open like this is to discover bugs earlier. I'd like there to be no (or at least far fewer) bugs when they get to RC.

Just search first on the existing [issues list](/dotnet/maui/issues) to see if anyone else has found a similar issue.
